lib/renovate.ts

```diff
@@ -13,7 +14,12 @@ proxy.bootstrap();
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 (async (): Promise<void> => {
-  process.exitCode = await globalWorker.start();
+  process.exitCode = await instrument(
+    'run',
+    async () => await globalWorker.start()
+  );
+  await telemetryShutdown(); //gracefully shutdown OpenTelemetry
```

Comment:
Currently this file is not used in the hosted app - we deep link to `renovateRepository()` and do initialization/reinitialization somewhat manually. Is there any implication if this `instrument()` is never called?

Reply:
It will not impact the functionality if tracing is not activated. If on the other hand you would want to use the traces then there is a impact. There would be no unifying span and therefore multiple traces for a single call of `renovateRepository()` are generated.